frontend/src/views/c-summary.vue

```diff
@@ -487,25 +498,51 @@ export default defineComponent({
       // create deep clone of this.repos to not modify the original content of this.repos
       // when merging groups
       const groups = this.hasMergedGroups() ? JSON.parse(JSON.stringify(this.repos)) as Array<Repo> : this.repos;
-      groups.forEach((repo) => {
-        const res: Array<User> = [];
+      const res: Array<User> = [];
 
-        // filtering
-        repo.users?.forEach((user) => {
-          if (this.isMatchSearchedUser(this.filterSearch, user)) {
-            this.getUserCommits(user, this.filterSinceDate, this.filterUntilDate);
-            if (this.filterTimeFrame === 'week') {
-              this.splitCommitsWeek(user, this.filterSinceDate, this.filterUntilDate);
+      if (this.filterSearch.startsWith('tag:')) {
+        const tagSearch = this.filterSearch.split('tag:')[1];
```

Comment:
Suggestion: slight naming adjustments
This could be renamed `searchedTags` (plural) to show users can search for multiple tags. You can make similar adjustments in the rest of your changes accordingly.